Added cloud cycle ability

JonathanHHenson opened the pull request:
You can cycle through clouds using the arrow keys now.

JonathanHHenson committed:
Merge remote-tracking branch 'CodingTrain/master'

Commit:
Use the arrow keys to cycle clouds

You can use the arrow keys to cycle through the clouds

Commit:
Fixed bug

Commit:
get SelectableOptions

Added ability to get selectable options from material select

Commit:
Fixed bug for reals this time

Commit:
Merge branch 'master' into master

meiamsome commented on custom_ui.js:
```diff
@@ -291,6 +291,26 @@ class MaterialSelect extends FormFields {
     }
 
     addSelectorListeners(){
```
Is this meant to be defined twice?

meiamsome commented:
It would be great if you could merge in the changes upstream again for me... Thanks

JonathanHHenson commented:
@meiamsome I got a little confused when trying to fix conflicts, I'll try to get rid of the weird duplicate

Commit:
Replaced all changes with master

meiamsome commented:
@TurqDevDesign, @AlcaDesign Review, please

Commit:
Merge branch 'master' into master

Commit:
Fixed properties to be fields

JonathanHHenson commented:
Everything should work without conflicting now

JonathanHHenson commented:
Just did a test and everything is working great :)

meiamsome commented:
Wait is there no changes to custom_ui now?

meiamsome commented:
It looks like it works fine - except that the selection menu's label stays the same.

JonathanHHenson commented:
@meiamsome, I don't need it anymore now

JonathanHHenson commented:
Oh, yes

JonathanHHenson commented:
I forgot to look into that

JonathanHHenson commented:
It's technically not broken though... :stuck_out_tongue:

JonathanHHenson commented:
Just setting up my environment, I'm tired of doing code editing on the github website

Commit:
Attemp to fix small bug

Hopefully makes selector value change to the correct value when cycling through clouds

AlcaDesign commented:
I suggest we/I/@TurqDevDesign add the ability to set the current option programmatically in `MaterialSelect`. You also need to take into account that I added sorting to the list so just iterating through the index will appear random rather than in the order that appears in the list.

Commit:
Undo last change

JonathanHHenson commented:
That would help a tonne!

TurqDevDesign commented:
I never got an email regarding this thread, I apologize that I haven't acknowledged it. Has anyone began implementing these changes?
